daycare/models.py

```diff
@@ -50,6 +49,12 @@ def create(email, username, password, name, description, price_per_day, capacity
 
         return new_daycare
 
+    def get_daycare_primary_image_url(self):
```

Comment:
maybe there should be a better method for a daycare user to decide when is his primary photo. 
that can wait, but worth thinking about.

Reply:
You're right. I thought about it, but for now, as we want to speed things up, this current state is good enough, and (hopefully) will be changed.

Reply:
We don't "hope", we do. :) please open an issue for it and tag it here.

Reply:
Created #93 :)